proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -14,8 +14,8 @@ service Service {
         option (google.api.http).post = "/cosmos/tx/v1beta1/simulate";
     }
     // Simulate simulates executing a transaction for estimating gas usage.
-    rpc TxByHash(TxByHashRequest) returns (TxByHashResponse) {
-        option (google.api.http).get = "/cosmos/tx/v1beta1/txByHash";
+    rpc GetTx(GetTxRequest) returns (GetTxResponse) {
+        option (google.api.http).get = "/cosmos/tx/v1beta1/getTx/{hash}";
```

Comment:
```suggestion
        option (google.api.http).get = "/cosmos/tx/v1beta1/tx/{hash}";
```

Reply:
for the sake of consistency between gateway URL and fq method name, maybe `Tx` or `TxByHash`? I believe that's what we did for modules, and app developers can overwrite anyways.